assets/figure_ids.nfc

```diff
@@ -438,7 +442,7 @@ Version: 1
 04d0: Frita
 04d1: Muffy
 04d2: Pietro
-04d3: Etoile
+04d3: Étoile
```

Comment:
That'll be a problem, the accept over the E.  Fairly sure these need to be ASCII